mods/configuring.md

```diff
@@ -72,6 +84,10 @@ The version of the mod; should follow [semver](https://semver.org), especially i
 
 The name of the mod's developer. Should be a single name, like "HJfod" or "Alk". If the mod has multiple developers, this should be a team name like "Geode Team".
 
+### `developers` 
+
+The name of the mod's developers. Replaces `developer`. Can be a single name, like \["HJfod"\] or \["Alk"\]. If the mod has multiple developers, this can be a team name like \["Geode Team"\], or you can list multiple developers names because, the `developers` property is a list/array. First developer listed is the main developer. If you have 3 or more developers listed, it will show `main developer + 2 more`. If you have 2 developers listed, it will show `Developer 1 & Developer 2`. If you list multiple developers, when someone clicks on your name(s), it will show the list of the developers, then you can click on one to see other mods they have made or are apart of.

```

Comment:
You shouldn't be putting team names in `developers`, just list out the team members